docs/4.1. Behaviour - Registration.md

```diff
@@ -79,15 +79,15 @@ If a network cable is temporarily unplugged, dependent on the time for which it
 
 The following error conditions describe likely scenarios encountered by Nodes when interacting with a Registration API. It is expected that all Node implementations handle these in a sensible fashion as described below.
 
-### Node Encounters HTTP 200 On First Registration
+### Node Encounters HTTP `200` On First Registration
 
-If a Node is restarted, its first action upon discovering a Registration API is to register its Node resource. On first registration with a Registration API this SHOULD result in a 201 (Created) HTTP response code. If a Node receives a 200 (OK) code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct 201 (Created) response code.
+If a Node is restarted, its first action upon discovering a Registration API is to register its Node resource. On first registration with a Registration API this SHOULD result in a `201` (Created) HTTP response code. If a Node receives a `200` (OK) code in this case, a previous record of the Node can be assumed to still exist within the network registry. In order to avoid the registry-held representation of the Node's resources from being out of sync with the Node's view, an HTTP `DELETE` SHOULD be performed in this situation to explicitly clear the registry of the Node and any sub-resources. A new Node registration after this point SHOULD result in the correct `201` (Created) response code.
 
-### Node Encounters HTTP 400 (Or Other Undocumented 4xx) On Registration
+### Node Encounters HTTP `400` (Or Other Undocumented `4xx`) On Registration
```

Comment:
We should check whether the anchor tags in the rendered docs change as a result of this (e.g. getting an extra '-' in them).

Reply:
Looks like it should be ok.  Local render gives

```
http://127.0.0.1:4000/is-04/branches/docs-language/docs/4.1._Behaviour_-_Registration.html#node-encounters-http-400-or-other-undocumented-4xx-on-registration
```